### PR TITLE
Make gRPC clients independent from OrbitGl

### PR DESCRIPTION
**Pull request by kisenshi**
### Move CrashManager and ProcessManager to new OrbitClientServices project.

Refactoring code related to gRPC clients so it is independent from OrbitGl. FramePointerValidatorClient depends on App.h so it is not possible to move as the other two.

### CODE REVIEW

File: OrbitClientServices/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+target_sources(OrbitClientServices PUBLIC 
+        include/OrbitClientServices/CrashManager.h
+        include/OrbitClientServices/ProcessManager.h)
+
+target_sources(OrbitClientServices PRIVATE
+        CrashManager.cpp
+        ProcessManager.cpp)
+
+target_link_libraries(OrbitClientServices PUBLIC 
+        OrbitCore
```
Comment (dimitry-) [inline OrbitClientServices/CMakeLists.txt:25]: + OrbitBase?

Comment (kisenshi) [reply OrbitClientServices/CMakeLists.txt:25]: I tried but it does not build if using OrbitBase instead

Comment (dimitry-) [reply OrbitClientServices/CMakeLists.txt:25]: github replaced + with dot. I meant in addition to OrbitCore

Comment (kisenshi) [reply OrbitClientServices/CMakeLists.txt:26]: Ahh, ok; done now. Thanks!

**Pull request merged**